code/index.html

```diff
@@ -4,18 +4,82 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
-    <!-- dont forget to add a css file and link it here! -->
-    <!-- If you're using google fonts, that should be added here -->
-  </head>
-  <body>
-    <!-- 
-      Write your code here inside the body to make it appear in the browser. 
-      Below is an example on how to use an image tag. Copy it and add it in the body, but outside the comment tag if you want to see how it looks like on the website. Then you can change the size and other styling for it in the css file.
-
-      <img src="example-recipe-background.png" alt="Banana Bread">
-
-    -->
-    <h1>Hello there! 👋🏼</h1>
-  </body>
-</html>
\ No newline at end of file
+    <title>Wanda's Portfolio</title>
+<link rel="stylesheet" type="text/css" href="style.css"> 
+<style>
+  @import url('https://fonts.googleapis.com/css2?family=Rubik&display=swap');
+  </style>
+</head>
+
+<body>
+<nav>
+  <a href="/work/">work</a> 
+  <a href="/about/">about</a> 
+  <a href="/contact/">contact</a>
+  </nav>
+
+  <div class="header">
+  <img src="./Images/olive-design-a.gif" alt="Logo Olive Design" width="700px" padding-bottom="-10px">
+  <h1>Welcome to Wanda's portfolio</h1>
+  </div>
```

Comment:
Indentation again, and try to keep all styling to the CSS file :) If you want a special styling for one specific element you can add a class name to it and style the class name (instead of styling the element directly in the HTML)

```
<div class="header">
  <img
    src="./Images/olive-design-a.gif"
    alt="Logo Olive Design"
    class="olive-image">
  <h1>Welcome to Wanda's portfolio</h1>
</div>
```

Reply:
Also, there's an element called `header` which you could've used here